albatross/core/parameter_handling_mixin.h

```diff
@@ -115,15 +116,22 @@ inline std::string pretty_priors(const ParameterStore &params) {
 
 inline std::string pretty_param_details(const ParameterStore &params) {
   std::ostringstream ss;
+
+  std::size_t max_name_length = 0;
```

Comment:
nit: Could you use https://en.cppreference.com/w/cpp/algorithm/max_element here?

Reply:
I think I'd have to create a vector of sizes first:
```
std::vector<std::size_t> sizes;
for (const auto &pair : params) {
    sizes.push_back(pair.first.size());
}
max_name_length = std::max_element(sizes.begin(), sizes.end());
```
or is there an easier way?

Reply:
I think you can pass in a comparator as a 3rd argument